Apps/WebClient/src/ClientApp/src/components/modal/verifySMS.vue

```diff
@@ -49,6 +50,8 @@ export default class VerifySMSComponent extends Vue {
     private isVisible = false;
     private isLoading = false;
     private isValid = false;
+    private timeoutMinutes = 5;
```

Comment:
class variable but only used on the method below. Also why 5?

Reply:
removed